tensorflow/compiler/mlir/tensorflow/transforms/unroll_batch_matmul.cc

```diff
@@ -117,28 +117,15 @@ std::vector<Value> ConvertTFBatchMatMulOp<BatchMatMulOpType>::sliceInput(
     auto reshape_op = createReshapeOp(value, {batch_size, num_rows, num_cols},
                                       element_type, loc, rewriter);
 
-    // Create a constant op for the split axis (=0)
-    auto split_dimension_type =
-        RankedTensorType::get({}, rewriter.getIntegerType(32));
-    auto split_dimension_attr = DenseElementsAttr::get(split_dimension_type, 0);
-    auto split_dimension_op = rewriter.create<TF::ConstOp>(
-        loc, split_dimension_type, split_dimension_attr);
-
     // Split along each batch.
```

Comment:
// Unpack ...

Reply:
Done in 843d9d64baa9bab72e8adb0de5b8644ff90ca53a